tests/unit/ReportUtilsTest.ts

```diff
@@ -1234,42 +1234,6 @@ describe('ReportUtils', () => {
             expect(shouldDisableThread(reportAction, reportID, false)).toBeTruthy();
         });
 
-        it('should disable on deleted and not-thread actions', () => {
-            const reportAction = {
-                message: [
-                    {
-                        translationKey: '',
-                        type: 'COMMENT',
-                        html: '',
-                        text: '',
-                        isEdited: true,
-                    },
-                ],
-                childVisibleActionCount: 1,
-            } as ReportAction;
-            expect(shouldDisableThread(reportAction, reportID, false)).toBeFalsy();
-
-            reportAction.childVisibleActionCount = 0;
-            expect(shouldDisableThread(reportAction, reportID, false)).toBeTruthy();
-        });
-
-        it('should disable on archived reports and not-thread actions', () => {
```

Comment:
This was a bad test because it wasn't actually testing archived reports. Archived reports would have required doing something with the report NVPs, not the status or state.

Then, the test is really only testing the `childVisibleActionCount`, which isn't great.

However, since the report wasn't actually archived, the logic falls into the same path as deleted threads since `reportAction` is missing the `message` property.